docker-compose-deploy.yaml

```diff
@@ -32,3 +33,46 @@ services:
     ports:
       - "5004:5004"
 
+  grafana:
+    build:
+      context: ./infra/grafana/
+      dockerfile: Dockerfile
+    container_name: grafana
+    ports:
+      - "3000:3000"
+    environment:
+      - GF_AUTH_ANONYMOUS_ENABLED=true
+      - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
```

Comment:
**issue (blocking):** We need safe secrets here, as logs can contain identifying or infrastructure-critical information.